data/services_links_example.yml

```diff
@@ -0,0 +1,22 @@
+---
+categories:
+- category_id: 1
```

Comment:
Are all the IDs in the config files useful for something?

Reply:
good point, no they are not